lib/stencilBundle.js

```diff
@@ -0,0 +1,321 @@
+require('colors');
+var MAX_SIZE_BUNDLE = 1048576 * 50; //50MB
+var _ = require('lodash');
+var Program = require('commander');
+var rr = require('recursive-readdir');
+var Archiver = require('archiver');
+var Async = require('async');
+var Crypto = require('crypto');
+var Fs = require('fs');
+var Jspm = require('jspm');
+var Path = require('path');
+var BundleValidator = require('../lib/bundleValidator');
+var Cycles = require('../lib/Cycles');
+var CssAssembler = require('../lib/cssAssembler');
+var LangAssembler = require('../lib/langAssembler');
+var TemplateAssembler = require('../lib/templateAssembler');
+var validator;
+
+function Bundle(themePath, themeConfig, rawConfig, options) {
+    validator = new BundleValidator(themePath, themeConfig, Program.marketplace !== true);
```

Comment:
I did that at first then found command runs as a singleton, but the feedback makes sense and reassures I had the correct approach before. I'll switch it back tomorrow